### PR TITLE
[Table] Columns headers fixed while scrolling

### PR DESCRIPTION
**Pull request by goelrohan6**
#1021

Table column's stay fixed while scrolling .

### CODE REVIEW

**Comment by hudsonfoo** [comment]
@goelrohan6 you're probably going to want to tweak my original example as there are some z-indexing issues when the table header overlaps it. I mainly just wanted to point you in a direction that would get you started.

**Comment by akhenry** [comment]
@goelrohan6 Thanks for pushing this. Before we can merge anything this is a Contributor License Agreement that needs to be completed. Take a look at the Contributing page on our website for instructions and a link - https://nasa.github.io/openmct/contributing/

It's also worth taking a look at the [contributing guide](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md) in our repository for some procedural and code requirements for contributions.

@charlesh88 Would you mind taking a look at these stylesheet changes?

**Comment by goelrohan6** [comment]
@akhenry I have completed the agreement and mailed .

**Comment by goelrohan6** [comment]
@akhenry I have filled the agreement and mailed.

File: platform/features/table/res/sass/table.scss [hunk 1]
```diff
@@ -26,7 +26,7 @@
     visibility: hidden;
     position: absolute;
 
-    //Add some padding to allow for decorations such as limits indicator
+    /*Add some padding to allow for decorations such as limits indicator*/
```
Comment (charlesh88) [inline platform/features/table/res/sass/table.scss]: // Is perfectly valid as a line comment in scss and is my preferred style

**Comment by charlesh88** [comment]
Hi, 
Thanks for the attention to this. Pulled this down and ran locally, and there are some minor issues noted as line comments. One major issue is that the use of `position: fixed` in the current implementation causes the thead to no longer be clipped when the table view area width is below the column's min width:

screen shot 2016-10-11 at 1 36 35 pm

I don't know if `position: fixed` will be a good solution path for this issue. This may require a more extensive re-architecting of the markup to solve.

**Comment by charlesh88** [comment]
As noted above, we'll be pursuing a solution here via an alternate methodology. Closing this.

**Pull request closed without merging**